modules/sdk-coin-eth/test/unit/transactionBuilder/flushTokens.ts

```diff
@@ -117,6 +129,27 @@ describe('Eth Transaction builder flush tokens', function () {
       txJson.data.should.startWith(flushForwarderTokensMethodId);
     });
 
+    it('a wallet flush forwarder transaction with forwarder Version 4', async () => {
+      const tx = await buildTransaction({
+        fee: {
+          fee: '10',
+          gasLimit: '1000',
+        },
+        counter: 0,
+        forwarderAddress: '0x53b8e91bb3b8f618b5f01004ef108f134f219573',
+        tokenAddress: '0xbcf935d206ca32929e1b887a07ed240f0d8ccd22',
+        contractAddress: '0x8f977e912ef500548a0c3be6ddde9899f1199b81',
+        forwarderVersion: 4,
+      });
+
+      tx.type.should.equal(TransactionType.FlushTokens);
+      const txJson = tx.toJson();
+      txJson.gasLimit.should.equal('1000');
+      txJson._type.should.equals(ETHTransactionType.LEGACY);
+      should.equal(txJson.nonce, 0);
+      txJson.data.should.startWith(flushForwarderTokensMethodIdV4);
```

Comment:
Please add a test to build txn from txhex and verity the toAddress and data

Reply:
@mullapudipruthvik Pls check now